src/nanoemoji/nanoemoji.py

```diff
@@ -94,13 +96,19 @@ def module_rule(mod_name, arg_pattern):
         + keep_glyph_names
         + " --output_file $out $in",
     )
+    if FLAGS.gen_svg_font_diffs:
+        nw.rule("svg2png", f"cairosvg -H {FLAGS.svg_font_diff_resolution}  -W {FLAGS.svg_font_diff_resolution} -f png -o $out $in")
     nw.newline()
 
 
 def picosvg(input_svg: str) -> str:
     return os.path.join("picosvg", os.path.basename(input_svg))
 
 
+def svg_png(input_svg: str) -> str:
+    return os.path.join("svg_png", os.path.basename(input_svg))
```

Comment:
you should change the file extension otherwise you end up saving a .png file (from cairosvg) with an incorrect .svg file extension

Reply:
Fixed.